custom/opm/reports.py

```diff
@@ -1112,7 +1116,7 @@ def rows():
                 for method, __, __, denom in self.model.method_map:
                     value = getattr(awc, method)
                     row.append(value)
-                    if denom != 'no_denom':
+                    if denom != 'no_denom' and denom != 'one':
```

Comment:
What's the aim of the 'one' denominator?  With 'no_denom', if an indicator returns 1 (as the truthy ones do, right?) in the column, doesn't it just sum up like any other column would?